tests/unit-tests/DataProviderTest.ts

```diff
@@ -64,4 +57,14 @@ describe('DataProvider', () => {
         }).finally(done);
         $timeout.flush();
     });
+
+    it("should not be able to page", () => {
+        var provider = new MyDataProvider($q);
+        expect(JSONForms.DataProviders.canPage(provider)).toBe(false);
+    });
+
+    it("should be able to filter", () => {
+        var provider = new MyDataProvider($q);
+        expect(JSONForms.DataProviders.canFilter(provider)).toBe(true);
+    });
```

Comment:
Please test at least the opposite case too. If you feel ambitious you can test all 4 cases.